.github/workflows/gradle.yml

```diff
@@ -150,6 +152,65 @@ jobs:
           attempt_limit: 3
           attempt_delay: 5000 # in ms
 
+  cdk-build:
+    needs: changes
+    runs-on: ubuntu-latest
+    # Because scheduled builds on master require us to skip the changes job. Use always() to force this to run on master.
+    if: needs.changes.outputs.cdk == 'true' || needs.changes.outputs.build == 'true' || (always() && github.ref == 'refs/heads/master')
+    name: "Airbyte CDK: Build"
+    timeout-minutes: 90
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+
+      - name: Cache Build Artifacts
+        uses: ./.github/actions/cache-build-artifacts
+        with:
+          cache-key: ${{ secrets.CACHE_VERSION }}
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "17"
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+
```

Comment:
"Connectors Base: Build" was adding "Install Pyenv" (introduced [here](https://github.com/airbytehq/airbyte/commit/683a67c023d316cdf4a2212ba6146529e3a42d25)) but it's not so clear to me why we would need this